docs/README.md

```diff
@@ -0,0 +1,41 @@
+## CPMA capabilities
+
+This documents list CPMA capabilities.
+
+---
+
+### Report
+
+Report consist of 2 parts:
+
+1. Pre-migration analytics that goes under "cluster" in json. It contains general information about: 
+  * Nodes - name, сpu usage, memory capacity, consumed memory, number of running pods, pod capacity, information that
+  indicates if node is master.
+  * Namespaces - name, latest change timestamp, container count, total cpu/memory usage.
```

Comment:
What about the items we list through namespaces ?

Reply:
Right! Missed it